src/Goerli-DssSpell.t.base.sol

```diff
@@ -1875,7 +1875,7 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
                 // Convert whole Dai units to expected RAD
                 uint256 normalizedTesthole = values.collaterals[ilk].dog_hole * RAD;
                 assertEq(hole, normalizedTesthole, string(abi.encodePacked("TestError/dog-hole-", ilk)));
-                assertTrue(hole == 0 || hole >= RAD && hole <= 50 * MILLION * RAD, string(abi.encodePacked("TestError/dog-hole-range-", ilk)));
+                assertTrue(hole == 0 || hole >= RAD && hole <= 65 * MILLION * RAD, string(abi.encodePacked("TestError/dog-hole-range-", ilk)));
```

Comment:
I think you can make it like 100M so we are fine for a while.

Reply:
Done in 7bbbdc0.